pxr/base/arch/fileSystem.cpp

```diff
@@ -1387,4 +1429,35 @@ void ArchFileAdvise(
 #endif
 }
 
+#if defined(ARCH_OS_WINDOWS)
+
+ARCH_API
+std::wstring ArchHandleLongWindowsPaths(const std::wstring& path)
```

Comment:
minor note, the pattern 

`ArchHandleLongWindowsPaths(ArchWindowsUtf8ToUtf16(path));`

is very common in this patch. What about overloading this function instead, with a const char* path overload to match the callsites?

Reply:
Good idea, done.